easybuild/easyblocks/g/geant4.py

```diff
@@ -64,6 +65,18 @@ def extra_options():
             'G4EMLOWVersion': [None, "G4EMLOW version", CUSTOM],
             'PhotonEvaporationVersion': [None, "PhotonEvaporation version", CUSTOM],
             'G4RadioactiveDecayVersion': [None, "G4RadioactiveDecay version", CUSTOM],
+            'enable_freetype': [False, "Build analysis library with freetype support", CUSTOM],
+            'enable_gdml': [False, "Build GDML support for the G4persistency library", CUSTOM],
+            'enable_inventor': [False, "Build OpenInventor visualization driver", CUSTOM],
+            'enable_inventorqt': [False, "Build OpenInventor Qt visualization driver", CUSTOM],
+            'enable_opengl': [False, "Build X11 OpenGL visualization driver", CUSTOM],
+            'enable_python': [False, "Install python bindings (Geant4Py)", CUSTOM],
+            'enable_qt5': [False, "Build Qt5 user interface and visualization drivers", CUSTOM],
+            'enable_raytracer': [False, "Build RayTracer visualization driver with X11 support", CUSTOM],
+            'enable_toolssg': [False, "Build ToolsSG visualization driver (with X11, motif, or Qt5 backend)", CUSTOM],
+            'enable_xm': [False, "Build motif user interface and visualization driver", CUSTOM],
+            'install_data': [False, "Download and install Geant4 datasests", CUSTOM],
+            'install_data_dir': [False, "Custom directory for Geant4 data files", CUSTOM],
```

Comment:
There is already a Geant4-data easyconfig that Geant4 should depend on that handles the data.
I suggest using that instead.

That way there is a consistent version of data installed for each version of Geant4

Reply:
good point, I somehow missed that